### ISSUE TITLE
IntelliJ region comments are marked as errors with klint

### ISSUE THREAD

**Issue by ipereziriarte** [shyiko/ktlint#278]
It looks like if the code contains [intellij's region comments](https://www.jetbrains.com/help/idea/using-code-editor.html#using_folding_comments), klint reports them with 
`"Missing space after // (cannot be auto-corrected)"`

Expected Behavior
Given a region comment like
```
//region LIFECYCLE
    fun onViewResumed() {
        runDisplayCountrySpecificViewsLogic()
    }
//endregion
```
Klint should not trigger any error

Region comments should be handled as an exception to the comment spacing rule.

Context
This is happening to me on Android Studio 3.1.4 and Detekt version 1.0.0.RC7-3

Maybe this issue is related to https://github.com/shyiko/ktlint/issues/273

Thoughts?

**Comment by shyiko** [comment]
Fixed in [0.29.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0290---2018-10-02).
